store/product.js

```diff
@@ -0,0 +1,27 @@
+import items from '../constants/shop';
+
+export default {
+  state() {
```

Comment:
VueX has to do with data that is shared between components. Here, this information is even already depicted in the route, so it does not need to be shared or even stored.

Reply:
Good point. Done